mutiny-core/src/node.rs

```diff
@@ -409,7 +409,16 @@ impl<S: MutinyStorage> Node<S> {
                         log_info!(logger, "Successfully retried spendable outputs");
                         persister.clear_failed_spendable_outputs()?;
                     }
-                    Err(e) => log_warn!(logger, "Failed to retry spendable outputs {e}"),
+                    Err(_) => {
+                        // retry them individually then delete
+                        // if there was only one we don't need to retry
```

Comment:
why?

Reply:
We were batching them all together, if one has already been swept but one not, then the whole thing would fail. This should help prevent that.